content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How Application Security works in Datadog
+kind: guide
+---
+
+## Overview
+
+Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities.
+
+APM records information about each HTTP request, these are referred to as traces. Application security uses the information APM is already collecting, and flags any suspicious requests or attack attempts as security signals. Depending on your settings and the severity of the attempt, you can receive notifications from Slack, email, or PagerDuty.
+
+Traditional Web Application Firewalls (WAFs) are usually deployed at the perimeter and have no context of the application behavior. For Application Security to be effective, it’s critical to be embedded in the application to get access to the data. Application Security provides WAF-like capabilities with additional application context and lower false positives.
+
+### Compatibility
+
+For Application Security to be compatible with your Datadog configuration, you must have APM enabled, and [send traces to Datadog][1]. It uses the same libraries used by the APM suite, so you don't need to deploy and maintain another library for this task. Steps to enable Application Security are specific to runtime language. Check to see if your language is supported in the [Application Security prerequisites][2].
+
+### Performance
```

Comment:
Good to mention that the additional processing is done platform-wise. All what is done on the service-side is matching the attack patterns